t/issues/t4182-resource-rerank.sh

```diff
@@ -3,6 +3,18 @@
 #  Ensure resource module resets ranks of R to match hostlist attribute
 #
 
+#  Do what flux start --test-hosts does, but using FLUX_TASK_RANK
```

Comment:
Commit message typo: "but the hostlist attribute will now used..." -> "now be used"